x-pack/plugins/cases/public/components/case_action_bar/index.test.tsx

```diff
@@ -61,18 +60,6 @@ describe('CaseActionBar', () => {
     );
   });
 
-  it('it should show the correct date', () => {
```

Comment:
This is covered by the status metrics tests.

Although one thing to note is that if we disable `lifespan` from the features we won't show the case creation date. I don't foresee needing to disable that feature though.

Reply:
It seems that the o11y does not want the metrics (https://github.com/elastic/kibana/pull/122608#issuecomment-1012927501). Let's discuss it offline.